test/securityHelper_test.go

```diff
@@ -0,0 +1,74 @@
+// define/securityHelpers
+package main
+
+import (
+	ed "crypto/ed25519"
+	"crypto/sha256"
+	b64 "encoding/base64"
+	"encoding/json"
+	"fmt"
+	"student20_pop/define"
+	"testing"
+	"math/rand"
```

Comment:
Maybe you're aware, maybe not:
> For random numbers suitable for security-sensitive work, see the crypto/rand package.
> https://golang.org/pkg/math/rand/

Reply:
Thanks, I've made the update but for this test I don't know if it has a significant difference

Reply:
Not in the tests. I'm just pointing that out in case you use that elsewhere :)